tools/fq.wdl

```diff
@@ -62,7 +62,7 @@ task fqlint {
     Float read2_size = size(read_two_fastq, "GiB")
 
     Int memory_gb = (
-        ceil((read1_size + read2_size) * 0.2) + 4 + modify_memory_gb
+        ceil((read1_size + read2_size) * 0.25) + 4 + modify_memory_gb
```

Comment:
That seems like a high memory requirement for this task. I assume you've encountered a failure. Looking at the list of validators, the only one that should require much memory is the duplicate name check.

Reply:
Yes it is definitely overkill for the majority of cases. But I've run into a _very_ small number of failures at the `0.2` scale (which is also more than what most cases need). So following the ethos for our default resource requirement settings (set defaults so we don't have failures in production/never need to think about it), I've been forced to set it this high.

I don't think we have much of a choice in the matter, unless we revise our resource policies. This is how much memory we need to assign so we guarantee no failures in production.

We could scale this back to a flat ~10gb which would be suitable for roughly 90% of cases? That's a gut estimate, no data to back it up. But then we'd need to override and set it higher to ~50gb for those corner cases. Do we want to deal with that? We have to keep in mind that there's no way to predict which samples are the problem samples ahead of time. Size is a very loose corollary.